en/components/expansion_panel.md

```diff
@@ -126,194 +104,149 @@ export class ExpansionPanelComponent {
 ```scss
-// in styles.scss
+// In expansion-styling.component.scss
+// Create expansion panel theme.
 $custom-panel-theme: igx-expansion-panel-theme(
+    // pass `$my-color-palette` palette.
     $palette: $my-color-palette,
-    $header-background: igx-color($my-color-palette, "primary", 200),
-    $header-focus-background: igx-color($my-color-palette, "primary", 300),
-    $header-title-color: igx-color($my-color-palette, "info"),
-    $header-icon-color: igx-color($my-color-palette, "info"),
-    $body-background: igx-color($my-color-palette, "secondary", 100),
-    $border-radius: 0.35em
+
+    // Styling parameters.
+    $header-background: igx-color($my-color-palette, "primary", 700),
+    $header-focus-background: igx-color($my-color-palette, "primary", 700),
+    $header-title-color: igx-color($my-color-palette, "secondary"),
+    $header-icon-color: igx-color($my-color-palette, "secondary"),
+    $body-background: igx-color($my-color-palette, "primary", 700),
+    $body-color: igx-color($my-color-palette, "secondary" 100),
+    $border-radius: .5
 );
 ```
+>[!NOTE]
+> To see all the available parameters for styling trough the [`theming`](themes/component-themes.md) engine check the [`API documentation`]({environment:sassApiUrl}/index.html#function-igx-expansion-panel-theme)
 
-### Applying
-All that's left is to properly scope our newly created theme.
-
-#### Globally
-In case you want this newly created `igx-expansion-panel theme` to be applied [`globally`](themes/component-themes.md#creating-themes) in your app, all that is needed is to include the theme in your app root style file and pass it to the [igx-expansion-panel mixin]({environment:sassApiUrl}/index.html#mixin-igx-expansion-panel):
+### Applying component theme
+Now to apply the component theme All that's left is to include `igx-expansion-panel` mixin and pass the `$custom-panel-theme` map.
 ```scss
-// in styles.scss
+// In expansion-styling.component.scss
 // Pass our custom-panel-theme to `igx-expansion-panel` mixin.
 @include igx-expansion-panel($custom-panel-theme);
 ```
-#### Scoped
-There may be a case where you want a particular `igx-expansion-panel` to be styled differently than the others in the app. This will require to use angular specific pseudo-class selectors like `:host`, `::ng-deep`, etc. Additionally, all of the above steps need to be moved from `styles.scss` to the `component.scss` file.
 
  >[!NOTE]
- >If the component is using an [`Emulated`](themes/component-themes.md#view-encapsulation) ViewEncapsulation, it is necessary to `penetrate` this encapsulation using `::ng-deep`.
+ >If the component is using an [`Emulated`](themes/component-themes.md#view-encapsulation) ViewEncapsulation, it is necessary to penetrate this encapsulation using `::ng-deep`.
  On the other side, in order to prevent our custom theme to leak to other components, be sure to include the `:host` selector before `::ng-deep`:
```

Comment:
```suggestion
 In order to prevent our custom theme from leaking into other components, be sure to include the `:host` selector before `::ng-deep`:
```